_posts/0000-01-02-compkatthegardener.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Passiflora coccinea
```

Comment:
She is not the Passiflora manicata.
![Passiflora  Coccinea-The Red Passionflower Vine-11082020 doc](https://user-images.githubusercontent.com/74166617/98501667-e4071f00-221d-11eb-89ee-dbb67ac8e634.jpg)